pkg/apis/settings/settings.go

```diff
@@ -65,37 +64,19 @@ func (*Settings) Inject(ctx context.Context, cm *v1.ConfigMap) (context.Context,
 }
 
 func (in *Settings) Validate() (err error) {
-	if in.BatchMaxDuration == nil {
+	if in.BatchMaxDuration == 0 {
```

Comment:
Not sure if we need the required check now that this value should always be set. Maybe we should just require users to set this value to at least a second, since I don't think a value lower than that makes a ton of sense.